src/main/java/nus/climods/model/ModelManager.java

```diff
@@ -23,65 +23,103 @@
  */
 public class ModelManager implements Model {
     private static final Logger logger = LogsCenter.getLogger(ModelManager.class);
+
     private final ModuleList moduleList;
-    private final UniqueUserModuleList uniqueUserModuleList;
+    private final UniqueUserModuleList userModuleList;
 
-    private final FilteredList<UserModule> filteredUserModules;
-    private final UserPrefs userPrefs;
     private final FilteredList<Module> filteredModuleList;
+    private final SortedList<Module> filteredAndSortedModuleList;
+    private final Comparator<? super Module> defaultModuleListComparator;
```

Comment:
The reason we need a `SortedList` is due to the `FilteredList` not having a way to change its ordering. This is not as clean as I wanted it to be